core/src/main/java/com/linecorp/armeria/client/Http2ResponseDecoder.java

```diff
@@ -285,7 +285,10 @@ public void onRstStreamRead(ChannelHandlerContext ctx, int streamId, long errorC
 
     @Override
     public void onPushPromiseRead(ChannelHandlerContext ctx, int streamId, int promisedStreamId,
-                                  Http2Headers headers, int padding) {}
+                                  Http2Headers headers, int padding) {
+        encoder.writeRstStream(ctx, promisedStreamId, Http2Error.REFUSED_STREAM.code(), ctx.newPromise());
+        ctx.flush();
+    }
```

Comment:
I'm a bit confused that these lines are not covered by tests. Is this a bug? 😅

Reply:
We don't need this code anymore because `PUSH_PROMISE` is disabled by default.
I left it in case that a certain server does not respect `SETTINGS_ENABLE_PUSH` disable well, but it would be better to revert these lines.

Reply:
By the way, the quality of Codecov report is pretty good. 😀

Reply:
Got it. 😄